opensir/models/post_regression.py

```diff
@@ -1,8 +1,86 @@
 """Post regression utilities"""
+from dataclasses import dataclass
 import numpy as np
+import matplotlib.pyplot as plt
 from sklearn.utils import resample
 
 
+@dataclass
+class PredictionResults:
+    """Class which stores model predictions,
+    based on block cross-validation """
+
+    pred: list  # List of out of sample predictions
+    n_obs_end: float  # Last in-sample observation
+    mse_avg: list  # MSE_avg after cross validation
+    n_avg: list  # Sample size of each MSE_avg
+    mse_seq: list  # List of lists of sequential MSE
+    mse_fc: list  # List of lists of forecasting for n-days MSE
+    p_cv: list  # List of parameters rolling-fitted through cross validation
+    # p_bt: list  # List of parameters sampled through bootstrap
+
+    def print_mse(self):
+        """Prints a summary of model predictive
+        performance measures for n_days forecasting
+        of the fitted variable """
```

Comment:
```suggestion
        """
        Prints a summary of model predictive
        performance measures for n_days forecasting
        of the fitted variable
        """
```

I am not sure if we are following any convention for comments. I thought black and pylint would have something to say about this.

Reply:
I agree. For some reason that didn't raise. I agree with your suggestion and fixed other mistakes.